addon/components/property-group.js

```diff
@@ -34,6 +34,18 @@ export default class SubmissionFormPropertyGroupComponent extends Component {
     return this.args.level || 1;
   }
 
+  get titleLevel() {
+    return `${this.level}`;
+  }
+
+  get titleSkin() {
+    return `${this.level + 1}`;
```

Comment:
I've used a getter here as well since that brings us 1 step closer to removing ember-math-helpers addon. (only `sub` remains I think).